front/src/routes/integration/all/zwavejs-ui/ZwaveJSUIDeviceBox.jsx

```diff
@@ -0,0 +1,214 @@
+import { Component } from 'preact';
+import { Text, Localizer, MarkupText } from 'preact-i18n';
+import cx from 'classnames';
+import get from 'get-value';
+
+import DeviceFeatures from '../../../../components/device/view/DeviceFeatures';
+
+import { connect } from 'unistore/preact';
+
+class ZwaveJSUIDeviceBox extends Component {
+  componentWillMount() {
+    this.setState({
+      device: this.props.device
+    });
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({
```

Comment:
Are you sure about this point?
On `componentWillReceiveProps`, I usually check if update is requested. Here, maybe you can check if `selector` changes before updating state.

Reply:
Copy-pasted from another integration :D The only way to properly check would be to do a deepequal between the two object, and i think it would properly be slower than just replacing the object like here

I agree it's not perfect, good enough for this MVP. (we are handling max 1 device aha, so)